activiti-spring-boot/spring-boot-samples/spring-boot-sample-migrate-users-groups-to-keycloack/docker/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM jboss/keycloak:3.2.0.Final
+MAINTAINER https://gitter.im/Activiti/Activiti7
```

Comment:
Don't think this file or directory is needed. The main.basedir is located using the directory maven plugin which should find the root level. So the test can just refer to the docker directory under root. See https://github.com/Activiti/Activiti/blob/master/activiti-spring-boot/spring-boot-starters/activiti-spring-boot-starter/pom.xml#L184 - that project doesn't have a docker subfolder, it refers to the one under root.

It looks like you've created a separate docker image just for these tests. But it's ok to re-use the existing image.